test/load.test.js

```diff
@@ -49,6 +49,22 @@ test('initialization failure with parameters', function(t) {
   t.end();
 });
 
+test('make sure that it works even if CSV is not loaded', function(t) {
+  segmentmap.getRouteSpeeds([90,91,92,93], (err, resp)=> {
+    if (err) { console.log(err); throw err; }
+    t.ok(resp, "Return results even if CSV is not loaded");
+    t.end();
+  });
+});
+
+test('dont load CSV and see if you get the correct response (4 INVALID_SPEEDs)', function(t) {
+  segmentmap.getRouteSpeeds([90,91,92,93], (err, resp)=> {
```

Comment:
I don't konw that it's guaranteed that `tape` runs the tests in the order shown in the file (it *probably* does, but it's not good to have tests depend on each other.

This test should construct a new `SpeedLookup()` object to call `getRouteSpeeds()` on - that way, you are guaranteed for it to be empty (see line 71 for an example below).

Some of the other tests are wrong because they make the same mistake (line 97), but that should be corrected as a separate fix.